.vitepress/config.mts

```diff
@@ -55,7 +57,7 @@ export default defineConfig({
         },
 
         footer: {
-          message: '<a href="/impress">Impressum und Kontakt</a>'
+          message: `<a href="${base}impress">Impressum und Kontakt</a>`
```

Comment:
In other places we already used vitepress' own api [withBase](https://vitepress.dev/reference/runtime-api#withbase) - but I don't know if it would work inside this config-file.

If not this approach is perfectly good 👍

Reply:
I gave it a try, but it looks like `withBase` does not work inside the config file:

config.mts
```
import { defineConfig, withBase } from 'vitepress'
```

resolves too 

```
build error:
file:///C:/Users/michael.prankl/git/opensource.muenchen.de/.vitepress/config.mts.timestamp-1697786185241-b9582ac12ae87.mjs:2
import { defineConfig, withBase } from "file:///C:/Users/michael.prankl/git/opensource.muenchen.de/node_modules/vitepress/dist/node/index.js";
                       ^^^^^^^^
SyntaxError: The requested module 'file:///C:/Users/michael.prankl/git/opensource.muenchen.de/node_modules/vitepress/dist/node/index.js' does not provide an export named 'withBase'
```